addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -113,6 +128,13 @@ public void unload() {
         super.unload();
     }
 
+    private AutomationPanel getAutomationPanel() {
+        if (automationPanel == null) {
+            automationPanel = new AutomationPanel(this);
```

Comment:
One question, the hook method defined above, does that get invoked multiple times?

Reply:
No, just once.